pkg/innerring/state.go

```diff
@@ -138,6 +147,28 @@ func (s *Server) voteForSidechainValidator(prm governance.VoteValidatorPrm) erro
 	return nil
 }
 
+func (s *Server) alreadyVoted(validatorsToVote keys.PublicKeys) (bool, error) {
+	// almost every part of NeoFS treats committee as validators list
+	validators, err := s.morphClient.Committee()
```

Comment:
Are you sure that it's safe to treat commettee as validators here? If these lists differ, then you may end up in a situation when you have to vote for some validator, but `alreadyVoted` method can return true for him (e.g. because he's a committee member, but not the validator yet).

Reply:
`len(committee) == len(validators)` is a wrong assumption which can be fixed easily. And you _must_ do `GetAccountState()`, you're just checking for a different thing otherwise.

Frankly, I was expecting a simple `GetAccountState()` (ok, a number of them) and comparison with the configured set. Then you don't even need to care about committee/validators list. You only need to care about votes here.